compiler/x/amd64/codegen/OMRRealRegister.hpp

```diff
@@ -267,7 +267,7 @@ class OMR_EXTENSIBLE RealRegister : public OMR::X86::RealRegister
 
    void setSourceRegisterFieldInEVEX(uint8_t *opcodeByte)
       {
-      uint8_t regNum = ((_fullRegisterBinaryEncodings[_registerNumber].needsRexForByte << 3) | _fullRegisterBinaryEncodings[_registerNumber].id);
+      uint8_t regNum = ((_fullRegisterBinaryEncodings[_registerNumber].needsRexPlusRXB << 3) | _fullRegisterBinaryEncodings[_registerNumber].id);
       uint8_t bits = 0;
       *opcodeByte &= 0x9F;
 
```

Comment:
I believe this will make sense when APX or xmm16-31 are implemented.

Reply:
OK.  I had forgotten that the full EVEX story wasn't completed.  Using names rather than raw mask bits would make this code more readable.  Nothing to fix now, but as more EVEX and eventual APX work is done I would like it overhauled.